src/planwise/component/scenarios.clj

```diff
@@ -44,6 +46,26 @@
   (-> (db-find-scenario (get-db store) {:id scenario-id})
       (update :changeset edn/read-string)))
 
+(defn- get-initial-providers
+  [store provider-set-id version filter-options]
+  (let [providers (providers-set/get-providers-with-coverage-in-region
+                   (:providers-set store) provider-set-id version filter-options)
+        select-fn (fn [{:keys [id name capacity lat lon]}]
+                    {:initial true :provider-id (str id)
```

Comment:
I don't think you need the `:initial` key here. Also, why convert the `id` into a string?